types/stylis/index.d.ts

```diff
@@ -6,20 +6,20 @@
 export as namespace stylis;
 
 export interface Element {
-    parent?: Element | undefined;
-    children?: Element[] | string | undefined;
-    root: Element;
+    parent: Element | null;
+    children: Element[] | string;
+    root: Element | null;
     type: string;
     props: string[] | string;
     value: string;
     length: number;
     return: string;
-    line?: number | undefined;
-    column?: number | undefined;
+    line: number;
+    column: number;
```

Comment:
`Element` is always created by the `node` function and all properties are always required (nothing is optional):
https://github.com/thysultan/stylis.js/blob/e6843c373ebcbbfade25ebcc23f540ed8508da0a/src/Tokenizer.js#L19-L21